lib/relation.ml

```diff
@@ -108,11 +108,7 @@ let add_missing_parents_of_siblings conf base indl =
 
 let dag_fam_list_of_ind_list indl =
   List.fold_left
-    (fun faml ind ->
-      let faml =
-        if List.mem ind.di_famc faml then faml else ind.di_famc :: faml
-      in
-      if List.mem ind.di_fams faml then faml else ind.di_fams :: faml)
+    (fun faml ind -> ind.di_fams :: ind.di_famc :: faml)
```

Comment:
We no longer need to check the absence of `ind.di_fams` and `ind.di_famc` in the accumulator before adding them to it?

Reply:
From what I gathered from the code It should not happen, but just in case I switched to using physical equality.